cpp/apidoc/tutorials/plasma.md

```diff
@@ -182,7 +182,7 @@ was written by the `Create` command.
 int64_t data_size = 100;
```

Comment:
We should perhaps turn this into an example we compile and include it in the Sphinx documentation